### ISSUE TITLE
Uncaught TypeError: Failed to set the 'value' property on 'AudioParam': The provided float value ...

### ISSUE THREAD

**Issue by albusdemens** [probablycorey/aural-coding#8]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.15
**System**: Mac OS X 10.10.5
**Thrown From**: [aural-coding](https://github.com/probablycorey/aural-coding) package, v1.0.0

### Stack Trace

Uncaught TypeError: Failed to set the 'value' property on 'AudioParam': The provided float value is non-finite.

```
At /Users/Alberto/.atom/packages/aural-coding/lib/aural-coding.coffee:140

TypeError: Failed to set the 'value' property on 'AudioParam': The provided float value is non-finite.
    at TypeError (native)
    at AuralCoding.module.exports.AuralCoding.noteOn (/Users/Alberto/.atom/packages/aural-coding/lib/aural-coding.coffee:140:27)
    at atom-workspace.<anonymous> (/Users/Alberto/.atom/packages/aural-coding/lib/aural-coding.coffee:42:24)
```

### Commands

```
     -9:57.5.0 pane:split-left (atom-text-editor.editor.is-focused)
     -0:01.8.0 core:backspace (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
aural-coding, v1.0.0
command-logger, v0.23.0
editor-stats, v0.17.0
language-latex, v0.6.1
language-matlab, v0.1.1
language-r, v0.2.0
linter-chktex, v1.1.0

# Dev
No dev packages
```

**Comment by capaj** [comment]
I got the same issue on debian 8.2

**Comment by ranman** [comment]
still getting this on osx

**Comment by FluxAugur** [comment]
+1 Getting this on osx

**Comment by swarut** [comment]
+1 on osx as well!

**Comment by probablycorey** [comment]
This should be fixed in the 1.0.1 release. I'm not sure what is causing this error. When it occurs it will now it log error messages to the console instead of showing an error.

If you see output from these log errors I would appreciate if you posted the results to an issue!

**Comment by swarut** [comment]
@probablycorey Thank, I just upgraded to 1.0.1 and it works fine :+1:
